### ISSUE TITLE
Use a more permissive license?

### ISSUE THREAD

**Issue by jhamman** [nco/pynco#41]
Currently pynco is licensed under GPL2. I am leaning toward changing this to a more permissive license such as MIT. I'm wondering if others (specifically @kwilcox @monocongo or @czender) have specific thoughts on how this may impact their use of the code.

**Comment by czender** [comment]
OK by me. NCO is GPL3 by the way.

**Comment by monocongo** [comment]
My use of NCO/pynco is pretty minimal, honestly. The [project](https://github.com/monocongo/climate_indices) I have that uses it is public domain for all practical purposes, with [Unlicense](https://unlicense.org/) as the "license". I've actually been meaning to revisit that since (I think) I need to add restrictions/notifications to properly acknowledge/adhere to the licensing terms of the code I leverage from other projects such as NCO/pynco.

TL;DR: The less restrictive the better for me.

**Comment by kwilcox** [comment]
I'm not distributing any software but also IANAL - the less restrictive the better! :+1: for MIT.
